.CI/build_all

```diff
@@ -11,7 +11,8 @@ if [ -z "$GH_TOKEN" ]; then
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
     conda install --yes --quiet conda-build-all
-    conda install --yes --quiet conda-build=1.20.0 jinja2 anaconda-client setuptools
+    conda install --yes --quiet conda-forge-build-setup
+    source "`conda info --root`/bin/run_conda_forge_build_setup"
```

Comment:
Surely the `conda info --root` is superfluous - conda is already on the PATH, so it should just find it...?